patches/server/0723-Fix-commands-from-signs-not-firing-command-events.patch

```diff
@@ -0,0 +1,103 @@
+From 0000000000000000000000000000000000000000 Mon Sep 17 00:00:00 2001
+From: Jake Potrebic <jake.m.potrebic@gmail.com>
+Date: Fri, 9 Jul 2021 13:50:48 -0700
+Subject: [PATCH] Fix commands from signs not firing command events
+
+This patch changes sign command logic so that `run_command` click events:
+  - are logged to the console
+  - fire PlayerCommandPreprocessEvent
+  - work with double-slash commands like `//wand`
+  - sends failure messages to the player who clicked the sign
```

Comment:
Why is this behaviour change necessary? Doesn't this result in message spam with existing setups which relied on that not being the case?

Reply:
Relied on the command throwing an exception?

Reply:
Is this really only for syntax errors/unknown commands? I was under the impression this would also catch cases where e.g. no entity would match a selector or a block could not be placed (a case Vanilla commands return "`false`")

Reply:
Yeah, it looks like it catches any exception. So I'll make this a config that defaults to the failure messages being hidden.